var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -62,132 +63,193 @@ class Mfem(Package):
             '841ea5cf58de6fae4de0f553b0e01ebaab9cd9c67fa821e8a715666ecf18fc57',
             url='http://goo.gl/xrScXn', extension='.tar.gz')
 
-    variant('metis', default=False, description='Activate support for metis')
-    variant('hypre', default=False, description='Activate support for hypre')
-    variant('suite-sparse', default=False,
-            description='Activate support for SuiteSparse')
-    variant('mpi', default=True, description='Activate support for MPI')
+    variant('mpi', default=True,
+        description='Enable MPI parallelism')
```

Comment:
Hmm. Well, I think its pretty obvious from the code. When one of them went over, I adjusted *all* of them for uniformity/consistency in look.